File: benchmark/python/services/db_service.py
import sqlite3

def get_user(user_id):
    conn = sqlite3.connect('database.db')
    cursor = conn.cursor()
    # SQL Injection [CWE-89]
    query = f"SELECT * FROM users WHERE id = '{user_id}'"
    cursor.execute(query)
    conn.close()

File: benchmark/ruby/services/db_service.rb
require 'sqlite3'

class DbService
  def self.get_user(user_id)
    db = SQLite3::Database.new "test.db"
    # SQL Injection [CWE-89]
    query = "SELECT * FROM users WHERE id = '#{user_id}'"
    db.execute(query)
  end
end
